src/videotoolbox.cs

```diff
@@ -608,6 +632,22 @@ interface VTEncodeFrameOptionKey {
 
 		[Field ("kVTEncodeFrameOptionKey_ForceKeyFrame")]
 		NSString ForceKeyFrame { get; } 
+
+		[TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Field ("kVTEncodeFrameOptionKey_AcknowledgedLTRTokens")]
+		NSString AcknowledgedLTRTokens { get; }
```

Comment:
`LTR` was lower-cased to `Ltr` further above:

```suggestion
		NSString AcknowledgedLtrTokens { get; }
```